zuul.d/run.yaml

```diff
@@ -1,6 +1,15 @@
 ---
 - hosts: all
   tasks:
+    - name: Activate uc-ui in inventory
+      become: yes
+      blockinfile:
+        path: "{{ zuul.project.src_dir }}/inventories/uc-engine"
+        marker: "# Uncomment the 2 following lines if you want to deploy wazo-ui too:"
```

Comment:
I don't know what is the good practice, but it seems a little bit weak to have a marker to a comment that doesn't deserve to be a marker

Reply:
it was to edit like a user would do it...